### PR TITLE
feat(performance): Display duration and ops percentage in breakdown tooltip

### PR DESCRIPTION
**Pull request by edwardgou-sentry**
Currently when hovering over ops durations in the Transaction Summary, we only display the name of the operation in the tooltip:
image

This change is to add value to these tooltips by also displaying the actual duration value and the percentage proportion of the total event duration
image
image
image

### CODE REVIEW

File: static/app/components/discover/transactionsList.tsx [hunk 1]
```diff
@@ -445,7 +447,18 @@ class TransactionsTable extends React.PureComponent<TableProps> {
             <GuideAnchor target="span_op_relative_breakdowns">
               <SortLink
                 align={align}
-                title={title}
+                title={
+                  title === t('operation duration') ? (
+                    <span>
+                      {title}
+                      <ExternalLink href="https://docs.sentry.io/product/performance/event-detail/#operations-breakdown">
```
Comment (shruthilayaj) [inline static/app/components/discover/transactionsList.tsx:454]: Hmm the pattern I notice everywhere else is that the StyledIconQuestion is used to indicate a tooltip, maybe there's a better icon to use or you could punt info about the ops breakdown in a tooltip instead?

**Review by shruthilayaj — CHANGES_REQUESTED** [review]

File: static/app/components/discover/transactionsList.tsx (hunk 1 above)
Comment (shruthilayaj) [reply static/app/components/discover/transactionsList.tsx:454]: Not sure if it's a big deal though

Comment (edwardgou-sentry) [reply static/app/components/discover/transactionsList.tsx:454]: @jeremybourque @doralchan thoughts?

Comment (edwardgou-sentry) [reply static/app/components/discover/transactionsList.tsx]: We decided to take take out the link for now and just go with the tooltip on the ops breakdown change 😄

**Review by shruthilayaj — APPROVED** [review]

File: static/app/utils/discover/fieldRenderers.tsx [hunk 2]
```diff
@@ -595,7 +595,21 @@ const spanOperationRelativeBreakdownRenderer = (
         }
         return (
           <div key={operationName} style={{width: toPercent(widthPercentage || 0)}}>
-            <Tooltip title={<div>{operationName}</div>} containerDisplayMode="block">
+            <Tooltip
+              title={
+                <div>
+                  <div>{`${operationName} ${formatPercentage(widthPercentage, 0)}`}</div>
+                  <div>
```
Comment (k-fish) [inline static/app/utils/discover/fieldRenderers.tsx:602]: Not sure you need a wrapping div? If you do, nevermind.

File: static/app/utils/discover/fieldRenderers.tsx [hunk 3]
```diff
@@ -595,7 +595,21 @@ const spanOperationRelativeBreakdownRenderer = (
...
+                    <Duration
```
Comment (k-fish) [inline static/app/utils/discover/fieldRenderers.tsx:603]: `PerformanceDuration`

**Review by k-fish — APPROVED** [review]

File: static/app/utils/discover/fieldRenderers.tsx (hunk 2 above)
Comment (edwardgou-sentry) [reply static/app/utils/discover/fieldRenderers.tsx:602]: Hey Kevan, using div to separate the tooltip text into 2 lines

**Pull request merged**